index.Rmd

```diff
@@ -1,105 +1,228 @@
+However, before getting into that step, I used some geological and engineering criteria to determine and selected the best landing/mining sites for the future rovers. The criteria used to select the landing/mining sites are: where are located minerals indicating the presence of water/water ice?, where do we have permanent sunlight to use as source of energy for processing the water and charging the rovers? Is the topography of the site suitable for the mobility of the rover to and from the processing center?... 
+For this project, the most important criteria I  considered is the geological criteria for question of time. I choosed and explored 9 sites as training sites and then selected among those sites Lyot crater"V00823007" which seem to be the best future landing/mining site. Within Lyot crater"V00823007", some locations show the presence of more volatiles materials than others. So the locations with abundant volatile materials are selected as future mining site and are then ploted on a Martian georeferenced globe.
 
 # Materials and methods
 
-[~ 200 words]
+I choosed 9 sites as training sites (Si polar swath"I00831006"; Lyot crater"V00823007"; Cerberus fossae"V00825005";Echus chasma"I00839002"; Sinus Meridiana west"I00849005"; Syrtis Major"I00853002"; Ophir planum"I00864002"; Marte vallis"I00868006; Eastern Meridiani"I00836002").
+For each training site, I downloaded and used Raster data. Those Raster data are Visible and infrared data in the “.TIFF” format and are combination of nine(9) or ten (10) bands according to Mars Odysee website. Each band is reflecting data in a specific wavelength. Even though all bands in each training site were already together in a single image in the website, I still re-added them together after using an image processing algorithm to correct the images in order to create a metadata file containing all information from each site. For Mars, I don't need to resize any band before adding them together because they all have the same size.I finally used the band ratio combination to determine the presence, abundance and distribution of water, water ice, ice or volatiles materials in each site I choosed as training site and from there I was able to see Lyot crater"V00823007" can be selected as the best future landing/mining site. I finally get the GPS coordinates of some locations within the Lyot crater"V00823007" site and plot those locations on a Martian globe using the package threejs.The locations shown on the Martian gloge which are within Lyot crater"V00823007" site are consider to be the future mining site because volatile materials are abundants in those locations compare to other location within Lyot crater"V00823007".
```

Comment:
This is not explained adequately.  The methods you use need to contained in this script (ideally) or explained with sufficient detail to reproduce the steps.  The line "still re-added them together after using an image processing algorithm to correct the images in order to create a metadata file containing all information from each site" is nonsensical.  Please revise/clarify.